src/screens/Login/Login.js

```diff
@@ -36,42 +36,49 @@ const Login = props => {
   };
 
   return (
-    <KeyboardAvoidingView style={styles.container} behavior="position" enabled>
-      <View style={styles.headerContainer}>
-        <View style={styles.infoImgContainer}>
-          <Image style={styles.infoImg} source={infoImg} />
+    <KeyboardAvoidingView style={styles.container} enabled>
```

Comment:
Could you please only remain Inputs in KeyboardAvoidingVeiw? when focusing in input all buttons are lifting up to bottom Input in current state

Reply:
It does not affects Login screen (inputs remain visible with keyboard), but this is necessary to match with designs.
![image](https://user-images.githubusercontent.com/15677991/58046764-e455bb00-7b4e-11e9-8eb7-6e1dc78dd5af.png)